compiler/rustc_target/src/callconv/riscv.rs

```diff
@@ -14,16 +14,18 @@ use crate::spec::HasTargetSpec;
 
 #[derive(Copy, Clone)]
 enum RegPassKind {
-    Float(Reg),
-    Integer(Reg),
+    /// (offset_from_start, type)
+    Float(Size, Reg),
+    /// (offset_from_start, type)
+    Integer(Size, Reg),
```

Comment:
Perhaps, if we must label these, it should instead be more like this?
```suggestion
    Float { offset: Size, ty: Reg }
    Integer { offset: Size, ty: Reg },
```

Reply:
I'm not terribly particular over the naming, just...